wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+            }
+
+            call fastqc.fastqc as indv_bamfqc { input:
+                inputfile = indv_mapping.sorted_bam,
+                default_location = "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/QC/FastQC",
+            }
+
+            call runspp.runspp as indv_runspp { input: bamfile = select_first([
+                indv_mapping.bklist_bam,
+                indv_mapping.sorted_bam,
+            ]) }
+
+            call bedtools.bamtobed as indv_bamtobed { input: bamfile = select_first([
+                indv_mapping.bklist_bam,
+                indv_mapping.sorted_bam,
+            ]) }
+
+            call util.evalstats as indv_summarystats { input:
+                fastq_type = "SEAseq Sample FASTQ",
+                bambed = indv_bamtobed.bedfile,
+                sppfile = indv_runspp.spp_out,
+                fastqczip = indv_fastqc.zipfile,
+                bamflag = indv_mapping.bam_stats,
+                rmdupflag = indv_mapping.mkdup_stats,
+                bkflag = indv_mapping.bklist_stats,
+                fastqmetrics = indv_bfs.metrics_out,
+                default_location = "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/QC/SummaryStats",
+            }
+        }  # end scatter (for each sample fastq)
+
+        # MERGE BAM FILES
+        # Execute analysis on merge bam file
+        # Analysis executed:
+        #   Merge BAM (if more than 1 fastq is provided)
+        #   FastQC on Merge BAM (AllMerge_<number>_mapped)
+
+        # merge bam files and perform fasTQC if more than one is provided
+        call util.mergehtml { input:
+            htmlfiles = indv_summarystats.xhtml,
+            txtfiles = indv_summarystats.textfile,
+            default_location = "SAMPLE",
+            outputfile = "AllMapped_" + length(fastqfiles) + "_seaseq-summary-stats.html",
+        }
+
+        call samtools.mergebam { input:
+            bamfiles = indv_mapping.sorted_bam,
+            metricsfiles = indv_bfs.metrics_out,
+            default_location = if defined(results_name) then results_name + "/BAM_files" else "AllMerge_" + length(indv_mapping.sorted_bam) + "_mapped" + "/BAM_files",
+            outputfile = if defined(results_name) then results_name + ".sorted.bam" else "AllMerge_" + length(fastqfiles) + "_mapped.sorted.bam",
+        }
+
+        call fastqc.fastqc as mergebamfqc { input:
+            inputfile = mergebam.mergebam,
+            default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/QC/FastQC",
+        }
+
+        call samtools.indexstats as mergeindexstats { input:
+            bamfile = mergebam.mergebam,
+            default_location = sub(basename(mergebam.mergebam), ".sorted.b.*$", "") + "/BAM_files",
+        }
+
+        if (defined(blacklist)) {
+            # remove blacklist regions
+            String string_blacklist = ""  #buffer to allow for blacklist optionality
```

Comment:
Should add spacing after `#`.

Reply:
I'll have to think about this one. Seems potentially dangerous. A naive approach would turn `#@ except` into `# @ except`, though that's pretty easy to fix/avoid. Maybe only insert a space if it matches the regex `#[a-zA-Z]`?

Reply:
I would insert it if it's the opening of a comment and it isn't another `#` or a `@` character.

Reply:
Here's the relevant lint rule: https://github.com/stjude-rust-labs/wdl/blob/21f3ae9458192c3574a399e162a4b256d34898f4/wdl-lint/src/rules/comment_whitespace.rs#L198-L211

Reply:
I'm not sure that's "correct". I have to think on this some more, but I'm feeling pretty confident we should only flag or fix the regex `/^#[a-zA-Z]/`. What about?
```
#------------------------------------------------------------
#| I don't like these comments but plenty of people use them |
#------------------------------------------------------------
```

Reply:
Maybe change the regex to include digits `/^#[a-zA-Z0-9]/`.

Reply:
Good call on the digits. But thinking some more, this may be another "opt-in" behavior. I don't think we should edit comments without a user explicitly asking us to, if that makes sense. I'm going to leave this unimplemented for now.